File: app/src/main/java/com/commit451/gitlab/api/GitLabRss.java
package com.commit451.gitlab.api;

import com.commit451.gitlab.model.rss.Feed;

import io.reactivex.Single;
import retrofit2.http.GET;
import retrofit2.http.Url;

public interface GitLabRss {

    @GET
    Single<Feed> getFeed(@Url String url);
}

File: app/src/main/java/com/commit451/gitlab/api/GitLabRss.kt
package com.commit451.gitlab.api

import com.commit451.gitlab.model.rss.Feed
import io.reactivex.rxjava3.core.Single
import retrofit2.Response
import retrofit2.http.GET
import retrofit2.http.Url

interface GitLabRss {

    @GET
    fun getFeed(@Url url: String): Single<Response<Feed>>
}
